src/shared/components/oncoprint/DataUtils.ts

```diff
@@ -379,6 +379,27 @@ function fillClinicalTrackDatum(
     return trackDatum;
 }
 
+function makeGetDataForCase(
```

Comment:
this is sort of crying out for a test.  i have a tool that should make it quite easy.  i'll merge and we can revisit.

Reply:
true, show me the tool when you get a chance